pkg/etcd/etcd_int_test.go

```diff
@@ -0,0 +1,120 @@
+package etcd_test
```

Comment:
Can you update this to be etcd or move it into another package called etcdtest but with no hyphens? This breaks standard conventions.

Reply:
This does not break standard convention. There is an exception specifically for black-box testing. See https://pkg.go.dev/cmd/go#hdr-Test_packages
```
Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary. 
```
There are examples of this in the go source code:
https://golang.org/src/cmd/go/note_test.go
https://golang.org/src/context/net_test.go

Reply:
Correct but that's an anti pattern that's been adopted that shouldn't be used. I'll find some articles as to why.